static/src/javascripts/projects/journalism/modules/submit-form.js

```diff
@@ -43,28 +44,52 @@ const showWaiting = cForm => {
     });
 };
 
+const readFile = file =>
+    new Promise(res => {
+        const reader = new FileReader();
+        reader.addEventListener(
+            'load',
+            () => {
+                // remove 'data:*/*;base64,' from the start of the string
+                const imageAsBase64 = reader.result
+                    .toString()
+                    .split('base64,')[1];
```

Comment:
Now that we get rid of the MIME type header, I'm wondering how FormStack will handle that.

Reply:
Have replaced this, can check in prod how formstack handle it..